pyproject.toml

```diff
@@ -24,6 +24,9 @@ classifiers = [
     "Topic :: Software Development :: Build Tools",
     "Typing :: Typed"
 ]
+dependencies = [
+  "importlib_metadata>=2.0; python_version<'3.10'",
+]
```

Comment:
```suggestion
```

should not be needed – please see https://github.com/scikit-build/cmake-python-distributions/pull/474#issuecomment-2020766395 :)